helpers/helpers_emba_dependency_check.sh

```diff
@@ -394,7 +394,7 @@ dependency_check()
     print_output "$ORANGE""To install all needed dependencies, run '""$NC""sudo ./installer.sh""$ORANGE""'." "no_log"
     print_output "$ORANGE""Learn more about the installation on the emba wiki: ""$NC""https://github.com/e-m-b-a/emba/wiki/installation\\n" "no_log"
 
-    if { [[ $ONLY_DEP -eq 0 ]] && [[ $FORCE -eq 0 ]] ;} || [[ $DEP_EXIT -gt 0 ]] ; then
+    if [[ $ONLY_DEP -eq 1 ]] || [[ $FORCE -eq 0 ]]; then
```

Comment:
Are you missing the DEP_EXIT now? Shouldn't it be like the following?
if [[ $ONLY_DEP -eq 1 ]] || [[ $FORCE -eq 0 ]] || [[ $DEP_EXIT -gt 0 ]]; then